commcare/src/main/resources/webapp/messages/messages.properties

```diff
@@ -65,6 +65,7 @@ commcare.cancel=Cancel
 commcare.all=All
 commcare.byDateRange=By Date Range
 commcare.formsToImport=Forms to Import
+commcare.verifyToSave=Verification is required
```

Comment:
@MichalKus Change hint message to something more clear like: Successful verification is required to save configuration